### PR TITLE
Test against PHP 8

### PR DESCRIPTION
**Pull request by snapshotpl**
Contains #68 changes

### CODE REVIEW

**Comment by abacaphiliac** [comment]
@snapshotpl i believe i would take [883ce02](https://github.com/enlitepro/enlite-monolog/pull/71/commits/883ce0258090056bff984047f3d3e37cd6286add) on its own in a separate PR. doing this before we migrate to laminas (and tagging before v3) would ensure that folks on v2 and php@8 have an option. what do you think? are you up for it?

**Comment by snapshotpl** [comment]
I we need first migrate to laminas - because it support php 8 and zend not. This PR will be small after merge #68 to master

**Comment by abacaphiliac** [comment]
@snapshotpl any chance i could get you to push a rebase so that it's easier to see what actually changed here? thanks in advance.

File: .github/workflows/coding-standards.yml [hunk 1]
```diff
@@ -23,7 +23,5 @@ jobs:
         uses: "ramsey/composer-install@v1"
         with:
           dependency-versions: "locked"
-      - name: "Coding Syntax"
-        run: "composer lint:syntax"
```
Comment (abacaphiliac) [inline .github/workflows/coding-standards.yml:27]: why was this dropped?

Comment (snapshotpl) [reply .github/workflows/coding-standards.yml]: Fixed by using replace package

**Comment by snapshotpl** [comment]
@abacaphiliac rebased

**Comment by abacaphiliac** [comment]
@snapshotpl great stuff, thank you for doing this. top shelf work.

**Comment by snapshotpl** [comment]
Ok so now waiting for release 😊

**Comment by abacaphiliac** [comment]
@snapshotpl any chance i could get you to validate compatibility of latest by installing from the dev branch before we tag? that way if there are any unexpected integration issues we can resolve them before 3.0.0 instead of in rapid-fire patches.

**Comment by snapshotpl** [comment]
Allready done and works in 100%

**Comment by snapshotpl** [comment]
@abacaphiliac current master works on PHP 8 couple days on production without any issues.

**Comment by abacaphiliac** [comment]
@snapshotpl thank you for your patience. [3.0.0](https://packagist.org/packages/enlitepro/enlite-monolog#3.0.0) has been published. thank you for all of your hard work and dedication to get this milestone over the finish line.

**Pull request merged**